packages/manager/modules/common-api/src/feature-availability/useFeatureAvailability.ts

```diff
@@ -0,0 +1,42 @@
+import { apiClient, ApiError } from '@ovh-ux/manager-core-api';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+
+export type UseFeatureAvailabilityResult<
+  T = Record<string, boolean>
+> = UseQueryResult<T, ApiError>;
+
+export const fetchFeatureAvailabilityData = async <T extends string[]>(
+  featureList: [...T],
+) => {
+  const result = await apiClient.aapi.get(
+    `/feature/${featureList.join(',')}/availability`,
+  );
+
+  const features = {} as Record<typeof featureList[number], boolean>;
+  featureList.forEach((feature) => {
+    features[feature] = feature in result.data ? result.data[feature] : false;
+  });
+
+  return features;
+};
+
+export const getFeatureAvailabilityQueryKey = <T extends string[]>(
+  featureList: [...T],
+) => [`feature-availability-${featureList.join('-')}`];
```

Comment:
```suggestion
) => ['feature-availability', ...featureList];
```
it would be easier to read in the reactQueryDevtool if it was in this form:
["feature-availability", "feature-1", "feature-2"]

Reply:
Yes good idea
done